framework/doc/content/source/meshgenerators/PatternedMeshGenerator.md

```diff
@@ -7,6 +7,13 @@
 The `PatternedMeshGenerator` object is similar to [TiledMeshGenerator](/TiledMeshGenerator.md). However, it is restricted to two dimensions and
 adds the ability to create a tile pattern from an arbitrary number of input meshes.
 
+!alert note
+The MeshGenerators provided in inputs are no longer required to have the same boundary ids for a given boundary name.
+In this case, boundary ids for all (cloned) input MeshGenerators are renamed to a common unused set of ids. The resulting stitched patterned mesh has boundary ids that are the same as the first entry of the passed inputs parameter (top left mesh in the grid).
```

Comment:
```suggestion
In this case, boundary ids for all (cloned) input MeshGenerators are first renamed to a common unused set of ids. Them the resulting stitched patterned mesh will be changed again to boundary ids that are the same as the first entry of the passed inputs parameter (top left mesh in the grid).
```
Last sentence does not work
See our input
```
    inputs = 'fmg fmg2'
    pattern = '1 0 0 0 0 0 0 0 0 0 0 0 0 0 ;
               0 1 1 0 0 0 0 0 0 0 0 1 1 0 ;
               0 1 1 1 0 0 0 0 0 0 1 1 1 0 ;
               0 1 0 1 1 0 0 0 0 1 1 0 1 0 ;
               0 1 0 0 1 1 0 0 1 1 0 0 1 0 ;
               0 1 0 0 0 1 1 1 1 0 0 0 1 0 ;
               0 1 0 0 0 0 1 1 0 0 0 0 1 0 ;
               0 1 0 0 0 0 0 0 0 0 0 0 1 0 ;
               0 1 0 0 0 0 0 0 0 0 0 0 1 0 ;
               0 1 0 0 0 0 0 0 0 0 0 0 1 0 ;
               0 1 0 0 0 0 0 0 0 0 0 0 1 0 ;
               0 0 0 0 0 0 0 0 0 0 0 0 0 0'
```

first input and top-left pattern could be not the same